Application/Client/src/app/_common/similarListing/components/similarListing.component.ts

```diff
@@ -15,33 +15,28 @@ export class SimilarListingComponent implements OnChanges{
 
   @Input()
   type;
-  
+
   @Output()
   similarListingLoaded:EventEmitter<string> = new EventEmitter();
-  
+
 
   public similarListing;
 
   constructor(private httpService:CService,private apiPath:apiPaths){
-    
+
   }
 
   ngOnChanges(changes: {[propKey: string]: SimpleChange}){
-    console.log("changes");
-    console.log(changes); 
     this.getSimilarListing();
   }
 
   getSimilarListing(){
-    console.log("type is",this.type)
-    //let filter = '';
     let pathUrl= '';
     let filter = this.type.split("-");
```

Comment:
remove this variables are are not used anywhere also scope of this variables are only in function